src/up.js

```diff
@@ -68,7 +68,11 @@ export default function up() {
           describe: 'The origin url to fetch pages content from.',
           type: 'string',
         })
-        .group(['pages-url', 'livereload', 'no-livereload', 'open', 'no-open', 'print-index'], 'Helix Pages Options')
+        .option('cache', {
+          describe: 'Path to local folder to cache the responses',
```

Comment:
```suggestion
        .option('alpha-cache', {
          describe: 'Path to local folder to cache the responses (note: this is an alpha feature, it may be removed without notice',
```